docker/release/opensuse13.2/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Docker image file that describes an OpenSUSE 13.2 (AKA harlequin) image with PowerShell installed from PowerShell Release
+FROM opensuse:harlequin
+LABEL maintaner Travis Plunk <travis.plunk@microsoft.com>
```

Comment:
Typo here, should be "maintainer" (also I though that was a specific Dockerfile command instead of a label).